pkg/game/game.go

```diff
@@ -21,10 +22,11 @@ type SerializedGame struct {
 	CurrentTile         tiles.Tile
 	ValidTilePlacements []elements.PlacedTile
 	CurrentPlayerID     elements.ID
-	Players             []elements.Player
+	Players             []elements.SerializedPlayer
 	PlayerCount         int
 	Tiles               []elements.PlacedTile
 	TileSet             tilesets.TileSet
+	BinaryTiles         []binarytiles.BinaryTile // contains info about all placed tiles, not placed tiles are equal to 0
```

Comment:
I wonder, isn't this a bit excessive? The API user can already call `to_bits()` on elements of the `Tiles` slice.